src/clockcache.c

```diff
@@ -2128,7 +2128,25 @@ clockcache_get_internal(clockcache   *cc,       // IN
    platform_status   status;
    uint64            start, elapsed;
 
-   debug_assert(allocator_get_ref(cc->al, base_addr) > 1);
+#if SPLINTER_DEBUG
+   uint8 extent_ref_count = allocator_get_ref(cc->al, base_addr);
+
+   // Dump allocated extents info for deeper debugging.
+   if (extent_ref_count <= 1) {
+      allocator_print_allocated(cc->al);
+   }
+   debug_assert((extent_ref_count > 1),
```

Comment:
Forcing this assertion to trip, by futzing this condition, the message will read like so, below:

```
splinter_test: SplinterDB performance test started with 1 tables
Assertion failed at src/clockcache.c:2138:clockcache_get_internal(): "(extent_ref_count > 2)". 
Attempt to get a buffer for page addr=4096, page type=5 ('superblock'), from extent addr=0, (extent number=0),
which is an unallocated extent, extent_ref_count=2.
Aborted (core dumped)
```